Update running ab tests (#1210)
vtex-apps/io-documentation: docs/en/Recipes/development/running-native-ab-testing.md

### Step 2 - Configuring the traffic and time

By agreeing to proceed with the test, you will need to answer the two following questions:

1. `What's the proportion of traffic initially directed to the master workspace?`. You must answer this question with any whole number between 0 and 10000. For example, if you answer `9000`, you'll set 90% of the traffic to the master workspace.

> ℹ️
>
> To avoid breaking your store and promote the changes from the new workspace in the safest way possible, we strongly recommend that you leave `90%` of traffic dedicated to the master workspace and the other 10% for the production workspace being tested.

> 2. `What's the amount of time respecting the restriction?`. This amount of time refers to the time (in hours) during which the proportion of traffic specified in the previous question will be kept constant. After that, the test will start to have the traffic proportions updated by the A/B testing system. The system analyzes each workspace's performance and sends more traffic to the best-performing ones. You can either:

Reviewer: ```suggestion
2. `What's the amount of time respecting the restriction?`. This is the amount of time (in hours) that the traffic proportion stated in the previous question remains constant. After this period, the A/B testing system begins to automatically update traffic proportions by measuring the performance of each workspace and sending more traffic to the best-performing one. There are two possible answers to this question:
```

>   - **Answer `0` to automatically proceed with the A/B test.** In this case, VTEX IO will automatically split your website traffic between workspaces, leaving 50% of your store's traffic with the master workspace while migrating the other 50% to the production workspace being tested. The platform will automatically balance traffic every 3 minutes according to the conversion rate that arises. This means that **the workspace with less conversion will progressively relay its traffic to the workspace with more conversion**. Notice that the test, although automatic, will not end by itself. It's important to evaluate and interpret the test status daily.

Reviewer: ```suggestion
  - **Answer `0` to automatically proceed with the A/B test.** In this case, VTEX IO will automatically split your website traffic between workspaces, routing 50% of your store's traffic to the master and the other 50% to the production workspace being tested. Following that, the platform will automatically balance traffic every three minutes based on the conversion rates. This means that traffic will be gradually routed from the workspace with the lowest conversion rate to the workspace with the highest conversion rate. It's important to note that the test does not end on its own. We also suggest that you evaluate the test results on a daily basis.
```

>   - **Answer with the number of hours you want to keep constant the proportion of traffic previously specified.** It's important for the test to extract the maximum amount of data during intense operational periods. At the same time, the test shouldn't overextend to be harmful to users navigating the workspace with the poorer experience.

Reviewer: ```suggestion
  - **Answer with the number of hours you want to keep constant the proportion of traffic previously specified.** During peak operational periods, it's critical for the test to extract as much data as possible. At the same time, the test shouldn't overextend and end up being harmful to users who are navigating the workspace with the poorest performance.
```

- **Start Date** - Date and time for the beginning of the test.
- **Running Time** - Test duration.
- **Probability B beats A** - Probability, in percentage points, that the production workspace is better for your store than the current master workspace. This calculation is based on the number of sessions and the number of sessions with completed sales. Notice: if the result of these metrics is greater than `10%`, the production workspace is able to become the winner.
- **Winner** - Workspace you chose as the winner. 

> ⚠️ 
> 
> The main results of the A/B test are aimed at scenarios where the platform automatically directed store traffic. While you can and should use A/B tests even if you manually direct the traffic, bear in mind that the numbers behind each result reflect an automatic segmentation according to each workspace experience.

**The best way to validate your A/B test workspace winner is to set a maximum conversion loss value according to your store operation size.**

> For example, you can set `0,0001%` as the conversion loss maximum value when starting your test. Then, the decision to end the test and declare a winner will be taken when either workspace reaches an `Expect Loss` result greater than `0,0001%`.

Reviewer: ```suggestion
For example, you can set a maximum conversion loss of `0,0001%` when starting your test. Then, when either workspace achieves an `Expect Loss` result greater than `0,0001%`, you should end the test and declare a winner.
```
